src/scripts/dungeons/Dungeon.ts

```diff
@@ -1306,6 +1306,59 @@ dungeonList['Berry Forest'] = new Dungeon('Berry Forest',
     ],
     1750, 29);
 
+dungeonList['New Island'] = new Dungeon('New Island',
+    [
+        new DungeonTrainer('Armored Mewtwo',
```

Comment:
```suggestion
        new DungeonTrainer('Armored',
```
Do this and change the filename to "Armored". No clue why it's not showing up, but this works. Maybe it's confusing the name with the pokemon name?